sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -389,7 +385,7 @@ export class BatchingReceiverLite {
           receiver.drain = true;
           receiver.addCredit(1);
```

Comment:
Not you..

Maybe just add a comment saying this triggers the drain which would also do the cleanup of removing the listeners.